doc/sphinx-guides/source/installation/r-rapache-tworavens.rst

```diff
@@ -238,7 +254,7 @@ b. Rename the resulting directory "dataexplore" ...
 
 ...and place it in the web root directory of your apache server. We'll assume ``/var/www/html/dataexplore`` in the examples below::
 
-        mv TwoRavens-a6869eb28693d6df529e7cb3888c40de5f302b66 /var/www/html/dataexplore
+        mv TwoRavens-dataverse-distribution /var/www/html/dataexplore
```

Comment:
I guess this is better than a6869eb? I guess we want to be able to change or fix the distributions of TwoRavens. Longer term, the TwoRavens team should tag their releases with version numbers.